test/integration/cloud-config-aws.yml.template

```diff
@@ -14,6 +14,8 @@
 aws_access_key: @ACCESS_KEY
 aws_secret_key: @SECRET_KEY
 security_token: @SECURITY_TOKEN
+aws_region: @REGION
```

Comment:
Since we are using AMIs in tests that require us-east-1 I don't think we can do this without breaking things right now.

Reply:
Updated link: https://github.com/ansible/ansible/blob/devel/test/integration/targets/elb_classic_lb/tasks/main.yml#L41

Reply:
So in that task `region` is a variable but `zones` are hardcoded? I don't think the problem is with my code here :)

Let me know what you want me to do about this - fixing that test would be the obvious fix for me (although I'll go with `{{ ec2_region }}a` and `{{ ec2_region }}b` as not all regions have four AZs

Reply:
I've updated the ELB tests to allow more flexible AZs, although I'm still using three AZs so the tests won't work anywhere with two AZs.

Reply:
If we can update all of our AWS tests to work in multiple regions (at least us-east-1 and us-east-2) that would have some benefits for CI. It would help with rate limiting issues, as we could run Python 2 tests us-east-1 and Python 3 tests in us-east-2.

Reply:
@mattclay I'm not aware of any test/integration/targets that now don't work in us-east-2 once this PR is accepted. Is it worth just trying it - happy to fix up any issues, I feel like I'm mostly (if not entirely) there now.

Reply:
I added AWS region support to ansible-test and fixed the .gitignore entries for the temporary cloud config files in a more general way with https://github.com/ansible/ansible/pull/30733.

Your changes to add region support to the tests should work now, although you'll want to rebase first.